features/crud.feature

```diff
@@ -16,7 +16,8 @@ Feature: Create-Retrieve-Update-Delete
           "value1",
           "value2"
         ]
-      }
+      },
+      "dummyPrice": 1.2
```

Comment:
It should throw an error here because you pass a int and it wants a string. It's a bug.